Makefile

```diff
@@ -167,14 +167,24 @@ conbench-on-minikube: build-conbench-container-image start-minikube
 	@make -s minikube-conbench-url
 
 
+# The `minikube ...service conbench-service --url` command returns immediately
+# on Linux, but is a long-running process on Darwin.
+# See https://github.com/conbench/conbench/issues/742.
+# Only run it on Linux.
 .PHONY: minikube-conbench-url
 minikube-conbench-url:
-	@CONBENCH_BASE_URL=$$(minikube --profile mk-conbench service conbench-service --url) && \
+	@if [[ "$$OSTYPE" == "darwin"* ]]; then \
+		echo "Darwin detected. Not running automatically, because it's long-running: " && \
+		echo "  minikube --profile mk-conbench service conbench-service --url" && \
```

Comment:
Would this URL be necessary to use `make db-populate`? If so, could you add small instructions here for using the output of this command (like you do in the linux branch)?

Reply:
I think it's OK to merge without this (it's additional value that we can add later). I agree that it's maybe nice. But I can't do that anymore today. Can you maybe do so, on this branch? :)

Reply:
Sure! I'll merge it afterwards.

Reply:
> Would this URL be necessary to use make db-populate?

Ah, and to your question: not necessary. In the sense that `minikube ..` is not the only way to construct this URL. Maybe you can get a bit of a feel which of both methods you like more, on your system:

- `kubectl port-forward ...` (as a long-running process)
- `minikube ... service ...` (as a long-running process)

On mac, both are long-running processes, and I don't know which of both I would like more. 

On linux, the latter is not long-running, which is why it adds value compared to  `kubectl port-forward ...`, which is why I started documenting this method in the first place.

Reply:
> Sure! I'll merge it afterwards.

Thanks!!

Reply:
I ended up recommending the `kubectl port-forward ...` way for Mac because it *doesn't* dynamically assign the port, and therefore we don't have to get the stdout of a long-running process. If a Mac user runs into problems here we can always take another look.

Reply:
good choice!